.github/workflows/labeler.yml

```
@@ -0,0 +1,25 @@
+name: Labeler
+
+on:
+  pull_request_target:
+    branches:
+      - main
+      - wip/*
+      - release/**
+    types: [ opened, synchronize, reopened ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)